### PR TITLE
New version: Ablaze.Floorp version 11.8.0

### PR DESCRIPTION
**Pull request by Exorcism0666**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/135135)

### CODE REVIEW

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping @Exorcism0666, I'll get a notification.
>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

File: manifests/a/Ablaze/Floorp/11.8.0/Ablaze.Floorp.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  - The design of the workspace icon has been adjusted to make it easier to see!
+    Thanks:@CutterKnife_
+  - We’ve changed the location of workspace settings and added a new workspace tab to the sidebar on Preferences page.
+  - We’ve changed the technology we use around tabs in PWA and Split View to improve performance.
+    Additionally, PWA detection behavior is now more accurate.
+  ❌ Fixed
+  - Resolves the issue where vertical tabs become unscrollable or stutter.
+  - @TheChilledBuffalo
+  - @ky3ow
+ReleaseNotesUrl: https://github.com/Floorp-Projects/Floorp/releases/tag/v11.8.0
```
Comment (Hibi-10000) [inline manifests/a/Ablaze/Floorp/11.8.0/Ablaze.Floorp.locale.en-US.yaml:55]:
```suggestion
ReleaseNotesUrl: https://blog.ablaze.one/3865/
```

File: manifests/a/Ablaze/Floorp/11.8.0/Ablaze.Floorp.locale.ja-JP.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+- browser
+- cross-platform
+- firefox-fork
+- foss
+- gecko
+- internet-browser
+- open-source
+- web
+- web-browser
+ReleaseNotesUrl: https://blog.ablaze.one/3760
```
Comment (Hibi-10000) [inline manifests/a/Ablaze/Floorp/11.8.0/Ablaze.Floorp.locale.ja-JP.yaml:27]:
```suggestion
ReleaseNotesUrl: https://blog.ablaze.one/3843/
```

**Review by Hibi-10000 — CHANGES_REQUESTED** [review]
The URL in `ReleaseNotesUrl` is wrong.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**